web/src/common/colors.ts

```diff
@@ -32,6 +34,8 @@ export let stats19ColorScale = commonQuintileColorScale.toReversed();
 
 export let poiColorScale = commonQuintileColorScale.toReversed();
 
+// Notice parts of the UI reverse these, to treat lowest ownership as the most
```

Comment:
I wasn't sure if it'd be cleaner to reverse here for most color scales / legends, and then un-reverse when we do something like `makeRamp` or `MetricBucket` that expects ascending limits.

Reply:
It's a bit complicated, but I think it's fine.